.travis.yml

```diff
@@ -1,18 +1,48 @@
+sudo: false
+
 language: cpp
 
 compiler:
    - gcc
-   - clang
+#   - clang
 
 os:
    - linux
-   - osx
+#   - osx
 
 env:
-   - DEBUG=YES MPI=YES TMPDIR=/tmp
-   - DEBUG=NO MPI=YES TMPDIR=/tmp
-   - DEBUG=YES MPI=NO
-   - DEBUG=NO MPI=NO
+   global:
+      - TMPDIR=/tmp
+   matrix:
+#      - DEBUG=YES
+#        MPI=YES
+#        CODECOV=NO
+#        MFEM_TEST_TARGET=check
+#      - DEBUG=NO
+#        MPI=YES
+#        CODECOV=NO
+#        MFEM_TEST_TARGET=test
+      - DEBUG=YES
+        MPI=NO
+        CODECOV=YES
+        MFEM_TEST_TARGET=test
+#      - DEBUG=NO
+#        MPI=NO
+#        MFEM_TEST_TARGET=test
+
+addons:
```

Comment:
Converted to container-based configuration.  Travis's non-container Ubuntu distro 'precise' has some security issues - for example the provided Python is too to use with 'pip', which is needed to install cpp-coveralls and pyyaml which I use to upload coverage results to coveralls.io.